src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -1707,49 +1704,16 @@ public Subject getSubject() {
      */
     @InterfaceAudience.Public
     @InterfaceStability.Evolving
-    public <T> T doAs(PrivilegedAction<T> action) {
```

Comment:
The diff looks weird -- it looks like this method is removed. Can you swap the order, so that `doAs` is first and the new `callAs` is second? That should look cleaner.

Reply:
Please swap the method order.